Python/perf_jit_trampoline.c

```diff
@@ -575,7 +576,7 @@ static void perf_map_jit_write_entry(void *state, const void *code_addr,
     ev.base.size = sizeof(ev) + (name_length+1) + size;
     ev.base.time_stamp = get_current_monotonic_ticks();
     ev.process_id = getpid();
-    ev.thread_id = gettid();
+    ev.thread_id = syscall(SYS_gettid);
```

Comment:
Perhaps it would make sense to leverage the existing thread API, e.g. `PyThread_get_thread_native_id()`?

Reply:
I did that first but that has a check first for `!initialized` and that was causing problems when fork happens.